src/classes/BGE_DataImportBatchEntry_CTRL_TEST.cls

```diff
@@ -0,0 +1,132 @@
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2018
+* @group BGE
+* @group-content //todo
+* @description Unit Tests for the Data Import Batch Entry component.
+*/
+
+@isTest(IsParallel=true)
+public with sharing class BGE_DataImportBatchEntry_CTRL_TEST {
+
+    /*********************************************************************************************************
+    * @description Sets up a new NPSP_Data_Import_Batch__c Record
+    * @return Id of the new NPSP_Data_Import_Batch__c
+    */
+    private static Id createDataImportBatch() {
+        DataImportBatch__c dataImportBatch = new DataImportBatch__c();
+        dataImportBatch.Name = 'Batch 1';
+
+        insert dataImportBatch;
+        return (Id)dataImportBatch.Id;
+    }
+
+    /*********************************************************************************************************
+    * @description Sets up a new DataImport__c Record
+    * @param recordName desired test new
+    * @param batchId ID of the related NPSP_Data_Import_Batch__c
+    * @return new DataImport__c record
+    */
+    private static List<DataImport__c> createDataImports(Integer numOfImports, Id batchId) {
+        List<DataImport__c> dataImports = new List<DataImport__c>();
+
+        for (Integer i=0; i<numOfImports; i++) {
+            DataImport__c dataImport = new DataImport__c();
+            dataImport.Account1_Name__c = 'Test Record ' + i;
+            dataImport.NPSP_Data_Import_Batch__c = batchId;
+            dataImports.add(dataImport);
+        }
+
+        insert dataImports;
+        return dataImports;
+    }
+
+    /*********************************************************************************************************
+    * @description Tests getDataImportModel() for a new record
+    */
+
+    public static testMethod void testGetNewDataImport() {
```

Comment:
Use @isTest instead of testMethod

Reply:
Why does github display `isTest` as `istest` ... ? ^ That t is totally capitalized.